src/reducers/index.js

```diff
@@ -0,0 +1,7 @@
+import { combineReducers } from 'redux';
+import booksReducer from './books';
+import filterReducer from './filter';
+
+const rootReducer = combineReducers({ theBooks: booksReducer, filter: filterReducer });
```

Comment:
- [ ] according to the requirements you should use `createStore` to create a store through the reducer created in the above step.